class UnwrapError(Exception):
    """Custom exception raised when an invalid 'unwrap' or 'unwrap_err' is called on a Result object."""
    pass
